### PR TITLE
FooConf 2023

### PR DESCRIPTION
**Pull request by teepeltola**
FooConf 2023 Helsinki, 1.2.2023

### CODE REVIEW

File: _posts/2023-02-01-fooconf-helsinki.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+Next in line for Teemu was a very nice speech about how a Legacy app went to serverless bar by **Sébastien Blanc** from Aiven. It was a humorous story featuring a 13 years old legacy app called Bob which wanted to go cloud and serverless. The story went through very humorously including funny characters like Join the table, Maria, Kafka river etc and ended up in an Aiven product demo, about serverless, Kafka, Kubernetes and autoscaling.
```
Comment (mattiuusitalo) [inline _posts/2023-02-01-fooconf-helsinki.md:51]: "humorous" used twice very close to eachother makes for a stuttering effect in the flow. Maybe sth like

"The story included funny characters like..." which would make for a more natural flow without repetition.

File: _posts/2023-02-01-fooconf-helsinki.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+![paste](/img/2023-fooconf/fooconf2023_5.jpg)
+_Legacy app went to serverless bar by Sébastien Blanc_
+
+Let’s do a thing and call it Foo by **Maaret Pyhäjärvi** was next in line for Markus. It was a speech about how to approach testing, which was defined as “Find (some of) what others may have missed”. The main takeaways from the speech were: testing is the responsibility of the whole team (even a team with designated testers), KNOW the domain where the application will be used (e.g. in clocks number 4 in roman numerals is IIII not IV) and choosing the right level of test coverage. Finally, the speech gave some useful insights from Maaret's own experiences in testing.
+
+After lunch, we both listened to the speech about Security by design by **Daniel Deogun** from OmegaPoint. It was about how to implicitly write safe software by going through the log4j vulnerabilities as an example. We learned that good design mitigates risks but not fully. Good discussion about what is safe and what is not.
```
Comment (mattiuusitalo) [inline _posts/2023-02-01-fooconf-helsinki.md:58]: *There was good discussion about what is safe and what is not.

^ would make for a complete sentence with a verb

File: _posts/2023-02-01-fooconf-helsinki.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+Markus listened to **Leif Åstrand** from Vaadin about Why and How to build a Collaborative UX. The speech was a presentation of the experiences of Vaadin on how they have tackled the issues when a collaborative UX is needed. Starting with use cases the speech went on to describe the architectural decisions to consider. The speech ended with a discussion about clustering in collaborative apps and how to implement a “distributed system without building a distributed system”. The speech provided good ideas for Markus which might come in handy shortly in his project work.
+
+Next in line for both Teemu and Markus was a speech called Devops that matters by **Melissa McKay** from JFrog. It was a discussion about starting with DevOps from a developer's point of view. Some of the common problems of dev teams and operations were discussed. It was emphasized that DevOps is a mindset from code to deployment and back. The most important takeaway from the speech was that silos should be eliminated! Many useful practical tips were given.
```
Comment (mattiuusitalo) [inline _posts/2023-02-01-fooconf-helsinki.md:67]: "Many useful practical tips were given."

I suggest using more active language like

"Melissa gave many useful practical tips."

File: _posts/2023-02-01-fooconf-helsinki.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+![paste](/img/2023-fooconf/fooconf2023_7.jpg)
+_Devops that matters by Melissa McKay_
+
+Then it was time for the last speech of the day, Thinking Architecturally by **Nate Schutta**. It was a truly humorous speech about software architecture. Nate is an experienced speaker and the speech provided some insights into everyday work in a very humorous way. There were some common themes with the first speech of the day by Tom Cools as Nate also emphasized the importance of constant learning and focusing on the core skills instead of learning all the new shiny technologies. We should learn from good and bad decisions. It was also repeated that we should be kind to ourselves.
```
Comment (mattiuusitalo) [inline _posts/2023-02-01-fooconf-helsinki.md:72]: "humorous" gets used a lot in this text. I suggest using something  like "playful", "funny", "hilarious" to mix things up a little bit.

**Review by mattiuusitalo — APPROVED** [review]
Some minor suggestions, to be applied if you like.

File: _posts/2023-02-01-fooconf-helsinki.md (hunk 1 above)
Comment (teepeltola) [reply _posts/2023-02-01-fooconf-helsinki.md]: Fixed, thanks!

File: _posts/2023-02-01-fooconf-helsinki.md (hunk 2 above)
Comment (teepeltola) [reply _posts/2023-02-01-fooconf-helsinki.md]: Fixed, thanks!

File: _posts/2023-02-01-fooconf-helsinki.md (hunk 4 above)
Comment (teepeltola) [reply _posts/2023-02-01-fooconf-helsinki.md:72]: Fixed, thanks!

File: _posts/2023-02-01-fooconf-helsinki.md [hunk 5]
```diff
@@ -48,14 +48,14 @@ After the first speech, we took some different tracks. Teemu listened to Cloud-n
...
 
-Next in line for Teemu was a very nice speech about how a Legacy app went to serverless bar by **Sébastien Blanc** from Aiven. It was a humorous story featuring a 13 years old legacy app called Bob which wanted to go cloud and serverless. The story went through very humorously including funny characters like Join the table, Maria, Kafka river etc and ended up in an Aiven product demo, about serverless, Kafka, Kubernetes and autoscaling.
+Next in line for Teemu was a very nice speech about how a Legacy app went to serverless bar by **Sébastien Blanc** from Aiven. It was a humorous story featuring a 13 years old legacy app called Bob which wanted to go cloud and serverless. The story including funny characters like Join the table, Maria, Kafka river etc and ended up in an Aiven product demo, about serverless, Kafka, Kubernetes and autoscaling.
```
Comment (mattiuusitalo) [inline _posts/2023-02-01-fooconf-helsinki.md:51]: "The story including funny characters" -> "The story inclu**ded** funny characters

Comment (teepeltola) [reply _posts/2023-02-01-fooconf-helsinki.md]: Thanks, fixed now :)

**Review by mattiuusitalo — APPROVED** [review]

**Pull request merged**